csrc/moe/marlin_moe_ops.cu

```diff
@@ -1872,7 +1849,6 @@ void marlin_mm_moe_f16i4(const void* A, const void* B, void* C,
     for (int m_block = 0; m_block < tot_m_blocks;
          m_block += 4 * exec_cfg.max_m_blocks) {
       // make it max possible value
```

Comment:
Should we remove the comment as well?

Reply:
Actually, I was a bit overzealous here -- restoring this so we can bring the unsupported shapes error message back to its former glory